backend/marina_app/admin.py

```diff
@@ -8,10 +8,21 @@ class StateAdmin(admin.ModelAdmin):
     list_display = ('name', 'abbreviation', 'population',
         'electronic_request', 'no_notary_required', 'no_fee', 'no_contact', 'no_witness_required', 'county_administered')
 
+class ImplementationGuidanceAdmin(admin.ModelAdmin):
+    list_display = ('name', 'question', 'why', 'quote', 'link')
 
+class IntroductionTextAdmin(admin.ModelAdmin):
```

Comment:
Should be a field under issue area

Reply:
I'm glad you suggested that! Will be sure to update

Reply:
Resolved on offshoot-implementation-components